### DISCUSSION TITLE
Disqus Comment Platform integration into the Relearn Theme

### DISCUSSION THREAD

**Discussion by dilipsundarraj1** [McShelby/hugo-theme-relearn#381] [General]
Hi Friends,

I have been trying to integrate the Disques Commenting Platform to my blog.

I have it working now, But for some reason the formatting is not right.

Followed these steps:

> 1) Created the disqus website 
> 2) Added this code in the custom-comments.html.
  > Note : Below snippet does not hold the exact name. Modified it for security purposes

```
<div id="disqus_thread"></div>
<script>
  (function() {
    var d = document, s = d.createElement('script');
    s.src = 'https://abc.disqus.com/embed.js';
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
  })();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
```
> Modified the config.toml file to include the **disqusShortname**
  > `disqusShortname = "abc"`

Output: 
Formatting seems to be odd, The comments section covers half of the page. 
Looking for help here to do it the right way. I am pretty sure someone have a solution for this .

![Screen Shot 2022-11-11 at 08 30 02](https://user-images.githubusercontent.com/26419531/201361315-50b8e3a7-8dd5-4536-8f30-fc3169fad20a.png)

**Comment by McShelby** [comment]
I am not familar with integrating discussions. I only can point to this documentation https://gohugo.io/content-management/comments/#configure-disqus.

If only the styling seems odd, you are free to style it to your liking in `custom-header.html`
